internal/connor/eq.go

```diff
@@ -43,15 +43,36 @@ func eq(condition, data any) (bool, error) {
 		}
 		return true, nil
 
+	case map[string]any:
+		d, ok := data.(map[string]any)
+		if !ok {
+			return false, nil
+		}
+		for k, v := range d {
+			m, err := eq(cn[k], v)
+			if err != nil {
+				return false, err
+			} else if !m {
+				return false, nil
+			}
+		}
+		return true, nil
+
 	case string:
 		if d, ok := data.(string); ok {
 			return d == cn, nil
 		}
 		return false, nil
 
+	case uint64:
```

Comment:
todo: `case uint64` and `case uint32` appear to be untested, and I expect they are dead code as json typically doesnt get parsed into unsigned types.  Please either delete them, or add integration tests covering them if you can.

Reply:
I removed them